packages/@vue/cli-plugin-typescript/lib/tslint.js

```diff
@@ -80,7 +80,11 @@ module.exports = function lint (args = {}, api, silent) {
     patchProgram(this.program)
   }
 
-  const tslintConfigPath = api.resolve('tslint.json')
+
+  const tslintConfigPath = tslint.Configuration.CONFIG_FILENAMES
+    .map(filename => api.resolve(filename))
+    .filter(file => !!fs.existsSync(file))
+    .pop();
```

Comment:
1. `fs.existsSync` returns a boolean, type casting is unnecessary.
2. Should return the first one in the array that exists. `.find()` is a better fit here.

Reply:
fixed with d5a3c96